docs/advanced-features/athena.md

```diff
@@ -1,91 +1,317 @@
+                    "Action": [
+                        "athena:*"
+                    ],
+                    "Resource": [
+                        "*"
+                    ]
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "glue:GetDatabase",
+                        "glue:GetDatabases",
+                        "glue:CreateTable",
+                        "glue:DeleteTable",
+                        "glue:UpdateTable",
+                        "glue:GetTable",
+                        "glue:GetTables",
+                    ],
+                    "Resource": [
+                        "*"
+                    ]
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:GetBucketLocation",
+                        "s3:GetObject",
+                        "s3:ListBucket",
+                        "s3:ListBucketMultipartUploads",
+                        "s3:ListMultipartUploadParts",
+                        "s3:AbortMultipartUpload",
+                        "s3:CreateBucket",
+                        "s3:PutObject",
+                        "s3:PutBucketPublicAccessBlock"
+                    ],
+                    "Resource": [
+                        ARN_ATHENA
+                    ]
+                },
+
+            ]
+}
+try: 
+    AthenaQuiltPolicy = IAM.create_policy(
+        PolicyName='AthenaQuiltAccess',
+        PolicyDocument=json.dumps(AthenaQuiltAccess),
+        Description='Minimal Athena Access policy for Quilt'
+    )
+    print(AthenaQuiltPolicy)
+except:
+    print("Policy AthenaQuiltAccess already exists")    
+```
+
+    Policy AthenaQuiltAccess already exists
+
+
+2. Attach this policy to your CloudFormation stack.
+ 
+This needs to be done manually by your AWS Administrator:
+
+    a. Go to the [CloudFormation console](https://console.aws.amazon.com/cloudformation)
+    b. Select the Quilt stack
+    c. Click "Update"
+    d. Add the above ARN to the "ManagedUserRoleExtraPolicies" field.
+    e. Save
+    
+3. Add that AWS policy as a Quilt catalog Policy
+
+This needs to be done manually by a Quilt Administrator:
+
+    a. Login to your Quilt instance at, e.g. https://quilt.mycompany.com
+    b. Click on "Admin Settings" in the upper right
+    c. Scroll down to the "Policies" section on the bottom
+    d. Click on the "+" to create a new Policy
+    e. Set Title to "AthenaQuiltAccess"
+    f. Check "Manually set ARN" and enter ARN of Athena policy.
+    f. Click "Create"
+    
+4. Attach that Policy to a (new) Quilt Role
+
+You cannot attach a policy to the "Custom" Roles, so you will usually need to first create a new Role:
+
+    a. From "Admin Settings", scroll to "Roles"
+    b. Click on the "+" to create a new Role
+    c. Set Name to e.g., "AthenaAccessRole"
+    d. Click on "No policies attached.  Attach a policy..."
```

Comment:
```suggestion
    d. Click on "No policies attached.  Attach a policy…"
```

Reply:
Need to modify this in the ipynb file

Reply:
Moved to a single-level of number under a header.

Reply:
Fixed.